CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # <!-- markdownlint-disable first-line-h1 no-inline-html -->
 
+## 2.9.3 (not release)
+
+BUG FIX:
+
+- `resource/vsphere_tag_category`: Updated code to not `ForceNew` for cardinality. 
+   This will stop the tag_category to not be removed.
+   ([#2263]https://github.com/hashicorp/terraform-provider-vsphere/pull/2263)
```

Comment:
```suggestion
- `r/vsphere_tag_category`: Updates resource not to `ForceNew` for cardinality. This will allow the `tag_category` to updated.
   [#2263](https://github.com/hashicorp/terraform-provider-vsphere/pull/2263)
```